etc/categories

```diff
@@ -184,8 +184,8 @@ WhatsApp|Internet
 Windows 10 Theme|Appearance
 Windows Flasher|Tools
 Windows Screensavers|Appearance
-Wine (x86)|Tools
-WPS Office|Editors
+Wine (x86)|Tools/Emulation
```

Comment:
It's pretty sneaky of you to define an app by the exact thing its name defies. (WINE = Wine Is Not Emulator)
Not that I have a problem with this though...

Reply:
I know lol, but wine itself IS running under box86 which is an emulator. so I felt like it was fitting regardless